<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBoot</class>
 <widget class="QDialog" name="FormBoot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>RotisSemiSans</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Start VMC</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>911</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabVMCSettings">
    <attribute name="title">
     <string>VMC settings</string>
    </attribute>
    <widget class="QListWidget" name="listVMCSettingsFiles">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>780</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonWriteSettings">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Write settings [USB-&gt;VMC]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReadSettings">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>340</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Read settings [VMC-&gt;USB]</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CPU firmware</string>
    </attribute>
    <widget class="QPushButton" name="buttonWriteCPU">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Write CPU firmware</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listCPUFiles">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>780</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>        GUI        </string>
    </attribute>
    <widget class="QListWidget" name="listGUIFiles">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>780</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonWriteGUI">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Write GUI [USB-&gt;VMC]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonReadGUI">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>340</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Read GUI [VMC-&gt;USB]</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Manual</string>
    </attribute>
    <widget class="QListWidget" name="listManualFiles">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>780</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonWriteManual">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Write manual</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Audit</string>
    </attribute>
    <widget class="QPushButton" name="buttonReadAudit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Read audit from VMC</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabML">
    <attribute name="title">
     <string>Multilang</string>
    </attribute>
    <widget class="QPushButton" name="btnWriteLang">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>331</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Write languages [USB -&gt; VMC]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReadLang">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>331</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Read languages [VMC -&gt; USB]</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RotisSemiSans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>labelStatus_text</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>RotisSemiSans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>labelWarning</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
